client/src/components/video.vue

```diff
@@ -107,13 +107,7 @@
           }
 
           &.request-control {
-            display: none;
-          }
-
-          @media (max-width: 768px) {
-            &.request-control {
-              display: inline-block;
-            }
```

Comment:
I also added `extraControls` param. Because when using neko, menu should not be shown on the right menu. Only when you have small screen. But in embed mode it should be visible all the time.